src/forward_kinematics.h

```diff
@@ -582,151 +582,167 @@ class RobotFK_Jacobian : public RobotFK_Fast
             {
                 while(mimic->getMimic() && mimic->getMimic() != joint_model)
                     mimic = mimic->getMimic();
-                joint_dependencies[mimic->getJointIndex()].push_back(joint_model);
+                joint_dependencies[static_cast<size_t>(mimic->getJointIndex())].push_back(joint_model);
             }
         }
-        tip_dependencies.resize(robot_model->getJointModelCount() * tip_count);
+        tip_dependencies.resize(robot_model_->getJointModelCount() * tip_count);
         for(auto& x : tip_dependencies)
             x = 0;
         for(size_t tip_index = 0; tip_index < tip_count; tip_index++)
         {
             for(auto* link_model = tip_links[tip_index]; link_model; link_model = link_model->getParentLinkModel())
             {
                 auto* joint_model = link_model->getParentJointModel();
-                tip_dependencies[joint_model->getJointIndex() * tip_count + tip_index] = 1;
+                tip_dependencies[static_cast<size_t>(joint_model->getJointIndex()) * tip_count + tip_index] = 1;
             }
         }
     }
-    void computeJacobian(const std::vector<size_t>& variable_indices, Eigen::MatrixXd& jacobian)
-    {
-        double step_size = 0.00001;
-        double half_step_size = step_size * 0.5;
-        double inv_step_size = 1.0 / step_size;
-        auto tip_count = tip_frames.size();
-        jacobian.resize(tip_count * 6, variable_indices.size());
-        for(size_t icol = 0; icol < variable_indices.size(); icol++)
-        {
-            for(size_t itip = 0; itip < tip_count; itip++)
-            {
-                jacobian(itip * 6 + 0, icol) = 0;
-                jacobian(itip * 6 + 1, icol) = 0;
-                jacobian(itip * 6 + 2, icol) = 0;
-                jacobian(itip * 6 + 3, icol) = 0;
-                jacobian(itip * 6 + 4, icol) = 0;
-                jacobian(itip * 6 + 5, icol) = 0;
-            }
+    void computeJacobian(const std::vector<size_t>& variable_indices,
+                       Eigen::MatrixXd& jacobian) {
+    double step_size = 0.00001;
+    // double half_step_size = step_size * 0.5;
```

Comment:
Let's not commit commented code. If we don't need this, just delete it.

Reply:
Not sure how I missed this. Addressed in [e9c01ee](https://github.com/PickNikRobotics/bio_ik/pull/1/commits/e9c01ee4ef00ced5d79bb788ab01a067e93e922c)